htlcswitch/link.go

```diff
@@ -431,6 +416,24 @@ func (l *channelLink) Start() error {
 			return fmt.Errorf("unable to trim circuits above "+
 				"local htlc index %d: %v", localHtlcIndex, err)
 		}
+
+		// Since the link is live, before we start the link we'll update
```

Comment:
👍

Reply:
Thoughts on making this synchronous instead of dispatching? As is, it's not guaranteed to register before making a state update